02_configure_host.sh

```diff
@@ -136,6 +136,7 @@ ansible-playbook \
     -e "baremetal_network_name=${BAREMETAL_NETWORK_NAME}" \
     -e "working_dir=$WORKING_DIR" \
     -e "num_masters=$NUM_MASTERS" \
+    -e "num_arbiters=$NUM_ARBITERS" \
```

Comment:
The playbook we're using here comes from https://github.com/metal3-io/metal3-dev-env/ which is an upstream project and thus does not (and can not) know about OCP-specific node types.

Reply:
I see, okay. I'm trying to make the changes Zane suggested, that hopefully should get rid of these additions.

Reply:
So I had to keep the additional flavor due to needing to alter the size of the arbiter node, but things are a bit more straight forward now.